core/ui_mixins/quick_panel.py

```diff
@@ -140,7 +148,9 @@ def show_paginated_panel(items, on_done, flags=None, selected_index=None, on_hig
             selected_index=selected_index,
             on_highlight=on_highlight,
             limit=limit,
-            next_message=next_message)
```

Comment:
I think I see now what is bothering me with this part, it's the tight coupling with the caller show_paginated_panel.

I think we can make this more readable if we move this "protection from None clause" to line 137 in show_paginated_panel, since it accepts kwargs explicitly defaulting to None.

PaginatedPanel itself can instead simply set any **kwargs given to it when initializing it, which should be mentioned in a docstring.

Please let me know how it sounds to you?